plutus-use-cases/test/Spec.hs

```diff
@@ -23,5 +24,6 @@ tests = localOption limit $ testGroup "use cases" [
     Spec.Crowdfunding.tests,
     Spec.Vesting.tests,
     Spec.Future.tests,
-    Spec.Game.tests
+    Spec.Game.tests,
+    Spec.Marlowe.tests
```

Comment:
Seems like it shouldn't be here.

Reply:
> (I'll try and review this a bit more later)

Thank you! I'll fix mentioned issues meanwhile